
// this file is automatically generated. you are highly discouraged from
// editing it manually
#include <zippylog/zippylog.hpp>

#include <zippylog/request_processor.pb.h>
#include <zippylog/message.pb.h>
#include <zippylog/core.pb.h>
#include <zippylog/protocol.pb.h>
#include <zippylog/device/store_watcher.pb.h>
#include <zippylog/device/server.pb.h>
#include <zippylog/protocol/response.pb.h>
#include <zippylog/protocol/request.pb.h>
// @@protoc_insertion_point(includes)

namespace zippylog {

    ZIPPYLOG_EXPORT void register_known_messages();
}
